GliaWidgets/SecureConversations/SecureConversations.ChatWithTranscriptModel.swift

```diff
@@ -180,43 +156,214 @@ extension SecureConversations {
     final class TranscriptModel: CommonEngagementModel {
         typealias ActionCallback = (Action) -> Void
         typealias DelegateCallback = (DelegateEvent) -> Void
-        enum Action {}
-        enum DelegateEvent {}
-        enum Event {
-            // TODO: MOB-1863
+        typealias Action = ChatViewModel.Action
+
+        enum DelegateEvent {
+            // TODO: MOB-1871
         }
 
+        typealias Event = ChatViewModel.Event
+
         var action: ActionCallback?
         var delegate: DelegateCallback?
 
         var engagementAction: EngagementViewModel.ActionCallback?
         var engagementDelegate: EngagementViewModel.DelegateCallback?
 
+        private let downloader: FileDownloader
+
+        private let isCustomCardSupported: Bool
```

Comment:
But we don't support custom cards in secure conversations, or are you future proofing?

Reply:
As I understand, chat transcript may return response cards. They are not intractable but visitor can see them. That is why this flag is here.